__tests__/app.test.js

```diff
@@ -312,12 +312,27 @@ describe("/api", () => {
       return request(app)
         .patch("/api/articles/1000")
         .send({
-          inc_votes: 5
+          inc_votes: 5,
         })
         .expect(404)
         .then((response) => {
           expect(response.body.msg).toBe("Article not found");
         });
     });
   });
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("DELETE 204: Should delete given comment by comment_id", () => {
+      return request(app).delete("/api/comments/1").send().expect(204);
+    });
+
+    test("DELETE 400: Should respond with an error if given an invalid comment id", () => {
+      return request(app)
+        .delete("/api/comments/not-an-id")
+        .send()
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
+        });
+    });
```

Comment:
Nice work checking the 400 for an invalid id. What about a valid id which doesn't exist? i.e. `/api/comments/32323` what would you expect this to respond with?